apps/client/src/components/Conclusion.js

```diff
@@ -1,20 +1,27 @@
-import { Button, Heading, Paragraph } from "@datapunt/asc-ui";
-import React, { Fragment } from "react";
-import { isIE, isMobile } from "react-device-detect";
+import { themeSpacing } from "@datapunt/asc-ui";
+import React from "react";
+import styled from "styled-components";
 
-import {
-  Alert,
-  ComponentWrapper,
-  HideForPrint,
-  PrintButton,
-  PrintOnly,
-} from "../atoms";
-import { Olo } from "../config";
-import { actions, eventNames, sections } from "../config/matomo";
+import { PrintOnly } from "../atoms";
+import { sections } from "../config/matomo";
 import withTracking from "../hoc/withTracking";
 import { sttrOutcomes } from "../sttr_client/models/checker";
-import ContactSentence from "./ContactSentence";
-import Markdown from "./Markdown";
+import { removeQuotes } from "../utils";
+import { NEED_CONTACT } from "../utils/test-ids";
+import {
+  ConclusionOutcome,
+  Disclaimer,
```

Comment:
Should Disclaimer live in `Conclusion` folder?

Reply:
Good point. Will change this.